### ISSUE TITLE
Massive stackoverflow with Minefactory Reloaded!

### ISSUE THREAD

**Issue by Unrepentant-Atheist** [Azanor/thaumcraft#1743]
Refering to https://github.com/CoFH/Feedback/issues/440

**Comment by Azanor** [comment]
Yeah, some mod must have recently added some TC compatibility that is breaking my aspect calc algorithm - there is a lot of these errors popping up lately.
